### PR TITLE
DOCS: improving documentation of add mesh link

### PR DESCRIPTION
**Pull request by anur7**

### CODE REVIEW

**Review by SebastienMorais — CHANGES_REQUESTED** [review]
Please remove the _unittest/.cov/xml file from the PR

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
As stated from another accound, please remove the `.cov` folder

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 1]
```diff
@@ -838,6 +839,18 @@ def add_mesh_link(
         ----------
 
         >>> oModule.EditSetup
+
+        Examples
+        --------
+        This example shows how to import the mesh from a source design solution.
```
Comment (gmalinve) [inline src/ansys/aedt/core/modules/solve_setup.py:845]: @anur7 for me this line is not needed.

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 2]
```diff
@@ -838,6 +839,18 @@ def add_mesh_link(
...
+        The target design is duplicated from the source design, and then the mesh
```
Comment (gmalinve) [inline src/ansys/aedt/core/modules/solve_setup.py:846]: @anur7 I'd put this line just before duplicating the design.

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 3]
```diff
@@ -838,6 +839,18 @@ def add_mesh_link(
...
+        is imported.
+
+        >>> from ansys.aedt.core import Maxwell3d
+        >>> m3d = Maxwell3d(design="source_design")
+        >>> m3d.create_setup(name="setup1")
+        >>> m3d.duplicate_design(name="source_design", save_after_duplicate=True)
+        >>> m3d.setups[0].add_mesh_link(design="source_design", solution="setup1 : LastAdaptive")
```
Comment (gmalinve) [inline src/ansys/aedt/core/modules/solve_setup.py:853]: @anur7 I'd put here an explanation that the active design in now the duplicated one and you are assigning the mesh link to its setup.
Also please check that _self.aedtapp.nominal_adaptive_ gives the same info in solution. This should give you back "setup1 : LastAdaptive".
Please add the release_desktop at the end.

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 4]
```diff
@@ -804,16 +804,17 @@ def add_mesh_link(
...
         design : str
-            Name of the design.
+            Name of the source design.
         solution : str, optional
-            Name of the solution in the format ``"name : solution_name"``.
+            Name of the source design solution in the format ``"name : solution_name"``.
             If ``None``, the default value is ``name : LastAdaptive``.
         parameters : dict, optional
-            Dictionary of the parameters.
+            Dictionary of the parameters of "mapping" variables from the source design.
```
Comment (gmalinve) [inline src/ansys/aedt/core/modules/solve_setup.py:817]:
```suggestion
            Dictionary of the "mapping" variables from the source design.
```

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 5]
```diff
@@ -804,16 +804,17 @@ def add_mesh_link(
         adapt_port=True,
     ):
         """Add a mesh link to another design.
+           Import mesh from a source design solution to the target design.
```
Comment (gmalinve) [inline src/ansys/aedt/core/modules/solve_setup.py:807]:
```suggestion
        """Import mesh from a source design solution to the target design.
```

**Review by gmalinve — CHANGES_REQUESTED** [review]

**Comment by gmalinve** [comment]
@anur7 please update the branch and reply to the discussion.

File: src/ansys/aedt/core/modules/solve_setup.py (hunk 3 above)
Comment (anur7) [reply src/ansys/aedt/core/modules/solve_setup.py:852]: yes, @gmalinve self.aedtapp.nominal_adaptive gives you "setup1 : LastAdaptive". Do you want me to subtitute the str with that comand?

Comment (gmalinve) [reply src/ansys/aedt/core/modules/solve_setup.py:852]: @anur7 yes, because this is what "solution" needs.

Comment (anur7) [reply src/ansys/aedt/core/modules/solve_setup.py]: @gmalinve I added your suggestion, please check

**Review by gmalinve — APPROVED** [review]

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**